docs/src/test.md

```diff
@@ -45,3 +45,72 @@ pytest -svv forge/test/mlir/test_ops.py::test_add
+
+To run all tests from test plan use `test_plan` method:
+
+```sh
+pytest -svv forge/test/operators/pytorch/test_all.py::test_plan
+```
+
+Full list of supported query criteria
+
+| Parameter          | Description                                                                                |
+| ------------------ | ------------------------------------------------------------------------------------------ |
+| OPERATORS          | List of operators                                                                          |
+
+To run single test based on a test id use `test_single` method with `TEST_ID` parameter:
+
+```sh
+TEST_ID='ge-FROM_HOST-None-(1, 2, 3, 4)-Float16_b-HiFi4' pytest -svv forge/test/operators/pytorch/test_all.py::test_single
+```
+
+| Parameter          | Description                                                                                |
+| ------------------ | ------------------------------------------------------------------------------------------ |
+| TEST_ID            | Id of a test containing test parameters                                                    |
+
+To see examples of all available tests use `test_unique` method:
+
+```sh
+pytest -svv forge/test/operators/pytorch/test_all.py::test_unique --collect-only
+```
+
+Full list of supported query criteria
+
+| Parameter          | Description                                                                                |
+| ------------------ | ------------------------------------------------------------------------------------------ |
+| OPERATORS          | List of operators                                                                          |
+
+To query subset of test plan based on a query criteria use `test_query` method:
+
+```sh
+OPERATORS=add,div FILTERS=HAS_DATA_FORMAT,QUICK DEV_DATA_FORMATS=Float16_b,Int8 MATH_FIDELITIES=HiFi4,HiFi3 RANGE=5 pytest -svv forge/test/operators/pytorch/test_all.py::test_query --collect-only
+```
+
+```sh
+OPERATORS=add,div FAILING_REASONS=DATA_MISMATCH,UNSUPPORTED_DATA_FORMAT SKIP_REASONS=FATAL_ERROR RANGE=5 pytest -svv forge/test/operators/pytorch/test_all.py::test_query --collect-only
+```
+
+```sh
+FAILING_REASONS=NOT_IMPLEMENTED INPUT_SOURCES=FROM_HOST pytest -svv forge/test/operators/pytorch/test_all.py::test_query --collect-only
+```
+
+Full list of supported query criteria
+
+| Parameter             | Description                                                                                   |
+| --------------------- | --------------------------------------------------------------------------------------------- |
+| OPERATORS             | List of operators                                                                             |
+| FILTERS               | List of labmda filters. See `test/operators/pytorch/test_all.py` for all values               |
```

Comment:
Next to the single sentence explanation, let's add few examples. And continuation should be as is now - where to find reference for all values.

Same comment applies for other examples as well :))

Reply:
Examples of usage and all available options are available via `print_query_docs`

Reply:
Documentation updated to include print_query_docs